### DISCUSSION TITLE
Multiple users

### DISCUSSION THREAD

**Discussion by CrysK** [joplin-vieweb/joplin-vieweb#56] [Q&A] [answered]
Hello together,

is it possible to use multiple users in joplin-vieweb?
I have created one admin and two user accounts in Django. But no matter where I log in, it always uses the same sync settings. 
Am I doing something wrong, or is this how it is supposed to be?

Thanks a lot

**Comment by gri38** [comment, marked as answer]
Hello.

The login feature is just for the notes not being public: need to log to read.
But it doesn't support multiple users.

I don't know if Joplin support (I think it was not when I wrote joplin-vieweb). I'll have a quite look...
...
So yes: Joplin desktop handles multiple profile. But Joplin Desktop is a... Desktop application ;-) only one instance runs at a time.
For joplin-vieweb: if you want several users to access their notes simultaneously, either I should run as many joplin terminal as user accounts.
Or the [data API](https://joplinapp.org/api/references/rest_api/) should offer to query for a specific profile, but it doesn't.

I think Joplin desktop simply restarts with loading the targeted user profile (so one directory per profile), but then you can access only the notes of that user. Which is good for a desktop application.

Joplin cloud handles users, but I don't know how.  

If you're interested you may start a discussion on joplin forum. If you answer here in this discussion with a link to joplin forum, I'll read it. 
I don't do it myself because a lack of time those times, but it may change ;-)
